### PR TITLE
Added 2 tests for CD servers: DefaultDefinitionDatabase + ListManagement

### PR DESCRIPTION
**Pull request by ArtemNizelnyk**
UnitTest is designed for DefaultDefinitionDatabase only.

ListManagement requires filling of the Sitecore.Info.IncludeFiles collection.

### CODE REVIEW

File: src/Sitecore.DiagnosticsTool.Tests.UnitTests/ECM/DefaultDefinitionDatabaseTests.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+using Sitecore.DiagnosticsTool.TestRunner.Base;
+using Sitecore.DiagnosticsTool.Tests.ECM;
+using Sitecore.DiagnosticsTool.Tests.UnitTestsHelper;
+using Sitecore.DiagnosticsTool.Tests.UnitTestsHelper.Context;
+using Sitecore.DiagnosticsTool.Tests.UnitTestsHelper.Resources;
+using Xunit;
+
+namespace Sitecore.DiagnosticsTool.Tests.UnitTests.ECM
+{
+    [TestClass]
```
Comment (AlenPelin) [inline src/Sitecore.DiagnosticsTool.Tests.UnitTests/ECM/DefaultDefinitionDatabaseTests.cs]: TestClass belongs to MSTest and it is not needed as [Fact] below belongs to Xunit and that [Fact] is sufficient enough.

File: src/Sitecore.DiagnosticsTool.Tests/ECM/DefaultDefinitionDatabase.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    public class DefaultDefinitionDatabase : Test
+    {
+        public override string Name => "Default Definition Database should be switched to 'web' on CD servers";
+
+        public override IEnumerable<Category> Categories => new[] { Category.Ecm };
+        public override IEnumerable<ServerRole> ServerRoles => new[] { ServerRole.ContentDelivery };
+
+        public override bool IsActual(IReadOnlyCollection<ServerRole> roles, ISitecoreVersion sitecoreVersion, ITestResourceContext data)
+        {
+            return sitecoreVersion.Major >= 8 && ServerRoles.Intersect(roles).Any();
```
Comment (AlenPelin) [inline src/Sitecore.DiagnosticsTool.Tests/ECM/DefaultDefinitionDatabase.cs]: No need to override `IsActual(ServerRole, ISitecoreVersion, ITestResourceContext)`, it should be enough to override `IsActual(ISitecoreVersion)` only.

File: src/Sitecore.DiagnosticsTool.Tests/ECM/DefaultDefinitionDatabase.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Sitecore.Diagnostics.Base;
+using Sitecore.Diagnostics.Objects;
+using Sitecore.DiagnosticsTool.Core.Categories;
+using Sitecore.DiagnosticsTool.Core.Tests;
+using Sitecore.DiagnosticsTool.Tests.ECM.Helpers;
```
Comment (AlenPelin) [inline src/Sitecore.DiagnosticsTool.Tests/ECM/DefaultDefinitionDatabase.cs]: Usings must be inside namespace.

File: src/Sitecore.DiagnosticsTool.Tests.UnitTests/ECM/DefaultDefinitionDatabaseTests.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Sitecore.Diagnostics.Objects;
+using Sitecore.DiagnosticsTool.Core.Categories;
+using Sitecore.DiagnosticsTool.TestRunner;
+using Sitecore.DiagnosticsTool.TestRunner.Base;
+using Sitecore.DiagnosticsTool.Tests.ECM;
+using Sitecore.DiagnosticsTool.Tests.UnitTestsHelper;
+using Sitecore.DiagnosticsTool.Tests.UnitTestsHelper.Context;
+using Sitecore.DiagnosticsTool.Tests.UnitTestsHelper.Resources;
+using Xunit;
```
Comment (AlenPelin) [inline src/Sitecore.DiagnosticsTool.Tests.UnitTests/ECM/DefaultDefinitionDatabaseTests.cs]: Usings must be inside namespace.

**Comment by ArtemNizelnyk** [comment]
I will create 2 correct Pull Requests instead of this one.

**Pull request closed without merging**